### PR TITLE
Updated docker-compose

### PR DESCRIPTION
**Pull request by clintonb**
- Exposing Elasticsearch ports
- Corrected course-discovery image tag
- Documented installation of elasticsearch-head

ECOM-2886

### CODE REVIEW

**Comment by clintonb** [comment]
@cpennington please review

File: docker-compose.yml [hunk 1]
```diff
@@ -10,6 +10,9 @@ es:
   container_name: es
   volumes:
     - /tmp/elasticsearch/data:/es-data
+  ports:
+    - "9200:9200"
+    - "9300:9300"
```
Comment (cpennington) [inline docker-compose.yml:15]: This is to allow a connection for debugging?

Comment (clintonb) [inline docker-compose.yml:15]: Debugging and/or accessing the elasticsearch-head UI.

File: docs/getting_started.rst [hunk 2]
```diff
@@ -84,3 +84,15 @@ services that it depends on.
...
+
+
+Install elasticsearch-head
+--------------------------
+Navigating Elasticsearch can be challenging if you don't have much experience with it. The `elasticsearch-head`_ plugin
+offers a web-based front end to help with this. The plugin can be installed in the `es` container with the command below.
+
+.. code-block:: bash
+
+    $ docker exec es /usr/share/elasticsearch/bin/plugin -install mobz/elasticsearch-head
```
Comment (cpennington) [inline docs/getting_started.rst:96]: Hm. One option, potentially, would be to add a Dockerfile to this repo that does this setup automatically. Another option would be to push that configuration into the `configuration` repo as part of our standard elasticsearch docker image.

Which one we do probably depends on whether we expect to use docker for production.

@e0d: Any thoughts?

**Pull request merged**